frontend/src/pages/dashboard/ExperimentsTable/ExperimentsTable.tsx

```diff
@@ -0,0 +1,201 @@
+    }),
+    []
+  );
+
+  const targetColumn = React.useMemo(
+    () => ({
+      Header: "Target",
+      accessor: (e: Experiment) => e.targetName,
+    }),
+    []
+  );
+
+  const ic50Column = React.useMemo(
+    () => ({
+      Header: "IC50",
+      accessor: (e: Experiment) => e.primarySubExperiment.item.result[2],
+    }),
+    []
+  );
+
+  const showCheckboxColumn = React.useMemo(
+    () => ({
+      id: "show",
+      accessor: (exp: Experiment) => (
+        <input
+          type="checkbox"
+          checked={selectedExperiments.has(exp.id)}
+          onChange={e => {
+            if (e.target.checked) {
+              addSubExperiment(exp.primarySubExperiment.id);
+            } else {
+              if (selectedSubexperiments.size === 1) {
+                setShowEntries("all");
+              }
+              removeSubExperiments(exp.subExperiments);
+            }
+          }}
+        />
+      ),
+    }),
+    [selectedSubexperiments, selectedExperiments, addSubExperiment, removeSubExperiments]
+  );
+
+  const minimizedColumns: Column<Experiment>[] = React.useMemo(
+    () => [compoundColumn, targetColumn, ic50Column, showCheckboxColumn],
+    [compoundColumn, targetColumn, ic50Column, showCheckboxColumn]
+  );
+
+  const expandedColumns: Column<Experiment>[] = React.useMemo(
+    () => [
+      compoundColumn,
+      targetColumn,
+      ic50Column,
+      {
+        Header: "Methodology",
+        accessor: (e: Experiment) => e.methodologyName,
+      },
+      {
+        Header: "Upload data",
+        accessor: (e: Experiment) => formatDateTimeDto(e.uploadDate),
+      },
+      showCheckboxColumn,
+      {
+        id: "actions",
+        accessor: () => <ContextActions actions={[]} />,
+      },
+    ],
+    [compoundColumn, targetColumn, ic50Column, showCheckboxColumn]
+  );
+
+  return (
+    <div className={cx("experimentsArea", className)}>
+      {experiments.length === 0 ? (
+        <EmptyPlaceholder title="No experiments match" />
+      ) : (
+        <>
+          <div className="experimentsArea__controlBtns">
+            <ShowEntriesSwitch
+              value={showEntries}
+              onChange={setShowEntries}
+              disabledSelected={selectedExperiments.size === 0}
+            />
+            <ExpandMinimizeButton targetSize="expanded" />
+          </div>
+          <div className="tableContainer experimentsArea__experimentsTableContainer">
+            <Table<Experiment>
+              small
+              columns={expTableSize === "minimized" ? minimizedColumns : expandedColumns}
+              data={
+                showEntries === "all"
+                  ? experiments
+                  : experiments.filter(e => selectedExperiments.has(e.id))
+              }
+              collapsible={e => (
+                <SuspenseSpinner>
+                  <ExperimentsCollapse experiment={e} />
+                </SuspenseSpinner>
+              )}
+            />
```

Comment:
![image](https://user-images.githubusercontent.com/25207759/114409811-82bfef00-9bb3-11eb-85f3-1aa51cac21e8.png)

Reply:
Fixed in this PR